.changes/next-release/feature-Logger-2c75802d.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
+  "category": "Logger",
+  "description": "filter out sensitive information from logger"
```

Comment:
Since this affects the developer experience, the description should probably be a bit longer. Maybe "Update the request logger to exclude sensitive input parameters from the message logged"